examples/jsm/nodes/shadernode/ShaderNodeBaseElements.js

```diff
@@ -0,0 +1,257 @@
+export const transformedNormalView = nodeImmutable( VarNode, normalView, 'TransformedNormalView' );
+
+export const viewMatrix = nodeProxy( Object3DNode, Object3DNode.VIEW_MATRIX );
+export const normalMatrix = nodeProxy( Object3DNode, Object3DNode.NORMAL_MATRIX );
+export const worldMatrix = nodeProxy( Object3DNode, Object3DNode.WORLD_MATRIX );
+export const position = nodeProxy( Object3DNode, Object3DNode.POSITION );
+export const viewPosition = nodeProxy( Object3DNode, Object3DNode.VIEW_POSITION );
+
+export const positionGeometry = nodeImmutable( PositionNode, PositionNode.GEOMETRY );
+export const positionLocal = nodeImmutable( PositionNode, PositionNode.LOCAL );
+export const positionWorld = nodeImmutable( PositionNode, PositionNode.WORLD );
+export const positionView = nodeImmutable( PositionNode, PositionNode.VIEW );
+export const positionViewDirection = nodeImmutable( PositionNode, PositionNode.VIEW_DIRECTION );
+
+export const texture = nodeProxy( TextureNode );
+export const sampler = ( texture ) => nodeObject( new ConvertNode( texture.isNode === true ? texture : new TextureNode( texture ), 'sampler' ) );
+export const uv = ( ...params ) => nodeObject( new UVNode( ...params ) );
+export const pointUV = nodeImmutable( PointUVNode );
+
+// math
+
+export const EPSILON = float( 1e-6 );
+export const INFINITY = float( 1e6 );
+
+export const cond = nodeProxy( CondNode );
+
+export const add = nodeProxy( OperatorNode, '+' );
+export const sub = nodeProxy( OperatorNode, '-' );
+export const mul = nodeProxy( OperatorNode, '*' );
+export const div = nodeProxy( OperatorNode, '/' );
+export const remainder = nodeProxy( OperatorNode, '%' );
+export const equal = nodeProxy( OperatorNode, '==' );
+export const assign = nodeProxy( OperatorNode, '=' );
+export const lessThan = nodeProxy( OperatorNode, '<' );
+export const greaterThan = nodeProxy( OperatorNode, '>' );
+export const lessThanEqual = nodeProxy( OperatorNode, '<=' );
+export const greaterThanEqual = nodeProxy( OperatorNode, '>=' );
+export const and = nodeProxy( OperatorNode, '&&' );
+export const or = nodeProxy( OperatorNode, '||' );
+export const xor = nodeProxy( OperatorNode, '^^' );
+export const bitAnd = nodeProxy( OperatorNode, '&' );
+export const bitOr = nodeProxy( OperatorNode, '|' );
+export const bitXor = nodeProxy( OperatorNode, '^' );
+export const shiftLeft = nodeProxy( OperatorNode, '<<' );
+export const shiftRight = nodeProxy( OperatorNode, '>>' );
+
+export const rad = nodeProxy( MathNode, MathNode.RAD );
+export const deg = nodeProxy( MathNode, MathNode.DEG );
+export const exp = nodeProxy( MathNode, MathNode.EXP );
+export const exp2 = nodeProxy( MathNode, MathNode.EXP2 );
+export const log = nodeProxy( MathNode, MathNode.LOG );
+export const log2 = nodeProxy( MathNode, MathNode.LOG2 );
+export const sqrt = nodeProxy( MathNode, MathNode.SQRT );
+export const invSqrt = nodeProxy( MathNode, MathNode.INV_SQRT );
```

Comment:
I think that in `ShaderNode` we can use the same functions name of `GLSL`?

Reply:
I think that it is better to follow the names in MathNode. We can rename `invSqrt` to `inverseSqrt` in both of them (MathNode and ShaderNodeBaseElements).

Reply:
In any case, it is better to change `MathNode` names, `GLSL` is more familiar, and I think we will have faster adoption the more similar it is.

Reply:
Maybe it would be better to also provide aliases like `inverseSqrt`, `inversesqrt`, and `invSqrt`?

Reply:
Let just keep `inverseSqrt` so? It looks good and is the same that WGSL too.

Reply:
Done.